react/src/views/Pages/LoginPage.jsx

```diff
@@ -42,19 +45,36 @@ class LoginPage extends React.Component {
   }
 
   handleLogin = () => {
+    // this.setState(
+    //   {
+    //     open_loginFail: true,
+    //   }
+    // )
     login(this.state.email, this.state.password).then(() => {
       // Handle successful login here
+      console.log("I am checking log in")
       this.redirectToHomepage()
     })
-    .catch(error => {
-      // Handle failed login here
-    })
+      .catch(error => {
+        // Handle failed login here
+        // console.log("Log in failed !!!")
+          // this.setState(
+          //   {
+          //     open_loginFail: true,
+          //   }
+          // )
```

Comment:
Can we **uncomment** this? The popup doesn't show up when I enter the wrong password.